ui/util/handleFlowError.ts

```diff
@@ -36,6 +36,11 @@ export const handleFlowError =
         // We need to re-authenticate to perform this action
         window.location.href = err.response.data.redirect_browser_to;
         return;
+      case "session_inactive":
+        // No session found, redirect the user to sign in
+        resetFlow(undefined);
+        await router.push("/login");
```

Comment:
I am not sure if this works right now. Currently the login logic is tied to hydra, if I try to go to `/ui/reset_password` right now without a session, I get redirected to this
![image](https://github.com/canonical/identity-platform-login-ui/assets/19745916/3b2a47cf-e957-42af-bb89-7bd0eaad8812)

The error on kratos is `The request was malformed or contained invalid parameters reason:the login_challenge parameter is present but empty`. The login-ui backend will need to change to handle this (probably on a different PR).

Reply:
Issue opened: https://github.com/canonical/identity-platform-login-ui/issues/247